codemp/rd-rend2/tr_world.cpp

```diff
@@ -569,13 +569,13 @@ static void R_RecursiveWorldNode( mnode_t *node, int planeBits, int dlightBits,
 		tr.pc.c_leafs++;
 
 		// add to z buffer bounds
-		tr.viewParms.visBounds[0][0] = std::min(node->mins[0], tr.viewParms.visBounds[0][0]);
-		tr.viewParms.visBounds[0][1] = std::min(node->mins[1], tr.viewParms.visBounds[0][1]);
-		tr.viewParms.visBounds[0][2] = std::min(node->mins[2], tr.viewParms.visBounds[0][2]);
+		tr.viewParms.visBounds[0][0] = MIN(node->mins[0], tr.viewParms.visBounds[0][0]);
```

Comment:
I can't compile the code otherwise, so it's needed for me somehow. I'm lazy to investigate why.